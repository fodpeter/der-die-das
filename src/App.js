import React from "react";
import "./App.css";
import Game from "./game";

const App = () => (
  <div className="App">
    <header className="App-header">
      <h1 className="App-title">Welcome to Der Die Das</h1>
    </header>
    <div className="App-intro">
      <Game />
    </div>
  </div>
);

export default App;
