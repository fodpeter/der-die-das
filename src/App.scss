$white: white;
$gray: #222;

.app {
  text-align: center;
}

.app-header {
  background-color: $gray;
  color: $white;
  padding: 20px;
}

.app-title {
  font-size: 1.5em;
}

.app-intro {
  font-size: large;
  margin: 0 auto;
  max-width: 600px;
}
