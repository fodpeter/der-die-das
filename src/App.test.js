import React from "react";
import { shallow } from "enzyme";
import App from "./App";

it("renders without crashing", () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});
