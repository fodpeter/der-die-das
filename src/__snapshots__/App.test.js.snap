// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="app-header"
>
          <h1
                    className="app-title"
          >
                    Welcome to Der Die Das
          </h1>
</header>,
        <div
          className="app-intro"
>
          <Connect(GameContainer) />
</div>,
      ],
      "className": "app",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="app-title"
>
            Welcome to Der Die Das
</h1>,
          "className": "app-header",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to Der Die Das",
            "className": "app-title",
          },
          "ref": null,
          "rendered": "Welcome to Der Die Das",
          "type": "h1",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Connect(GameContainer) />,
          "className": "app-intro",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="app-header"
>
            <h1
                        className="app-title"
            >
                        Welcome to Der Die Das
            </h1>
</header>,
          <div
            className="app-intro"
>
            <Connect(GameContainer) />
</div>,
        ],
        "className": "app",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="app-title"
>
              Welcome to Der Die Das
</h1>,
            "className": "app-header",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome to Der Die Das",
              "className": "app-title",
            },
            "ref": null,
            "rendered": "Welcome to Der Die Das",
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(GameContainer) />,
            "className": "app-intro",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
